backstage/plugins/knative-event-mesh-backend/src/providers/knativeEventMeshProcessor.ts

```diff
@@ -14,6 +14,7 @@ import {TypeKnativeEvent} from "./types";
 export class KnativeEventMeshProcessor implements CatalogProcessor {
     private readonly catalogApi:CatalogClient;
     private readonly logger:Logger;
+    private readonly queryEntityPageLimit:number = 10000;
```

Comment:
For making this testable, can you pass the `queryEntityPageLimit` in the constructor? It can be an optional argument, defaulting to 10000.